### PR TITLE
Feature/22396 various improvements

### PR DESCRIPTION
**Pull request by cmoesgaard**
Fixes:

https://redmine.magenta-aps.dk/issues/29280
https://redmine.magenta-aps.dk/issues/29278
https://redmine.magenta-aps.dk/issues/29277
https://redmine.magenta-aps.dk/issues/29276
https://redmine.magenta-aps.dk/issues/29274
https://redmine.magenta-aps.dk/issues/29272

### CODE REVIEW

File: frontend/src/store/modules/searchResults.js [hunk 1]
```diff
@@ -64,6 +64,9 @@ const actions = {
         results.push(JSON.parse(filteredResults[item].document))
       }
     }
+    results.sort(function (a, b) {
+      return a.name.localeCompare(b.name, 'da')
```
Comment (danchr) [inline frontend/src/store/modules/searchResults.js]: Can't SOLR do this for you?

File: frontend/src/components/VTreeView.vue [hunk 2]
```diff
@@ -81,6 +81,20 @@ export default {
               tree.push(node)
             }
           })
+
+          let sortTree = tree => {
+            tree.forEach(node => {
+              let children = node.children
+              if (children) {
+                children.sort((a, b) => {
+                  return a.name.localeCompare(b.name, 'da')
```
Comment (danchr) [inline frontend/src/components/VTreeView.vue]: Likewise; perhaps we should do this server-side?

File: frontend/src/store/modules/searchResults.js (hunk 1 above)
Comment (cmoesgaard) [reply frontend/src/store/modules/searchResults.js]: Good question. I'll look into it.

Comment (cmoesgaard) [reply frontend/src/store/modules/searchResults.js]: It requires more work than it's worth. Sorting in UI is efficient enough. I've added comments with explanations.

**Pull request merged**